ed448/Cargo.toml

```diff
@@ -0,0 +1,45 @@
+[package]
+authors = ["RustCrypto Developers"]
+categories = ["cryptography"]
+description = """A pure-Rust implementation of Ed448 and Curve448 and Decaf.
+This crate also includes signing and verifying of Ed448 signatures.
+"""
+documentation = "https://docs.rs/ed448-goldilocks"
+exclude = [".gitignore", ".github/*"]
+edition = "2021"
+homepage = "https://docs.rs/ed448-goldilocks/"
+keywords = ["cryptography", "decaf", "ed448", "ed448-goldilocks"]
+license = "BSD-3-Clause"
+name = "ed448"
```

Comment:
If we keep the name as `ed448-goldilocks`, it will [allow the `ed448` name to be used for the signature abstraction crate](https://github.com/RustCrypto/signatures/blob/master/ed448/Cargo.toml#L2)

(and that also ensures existing users get updates automatically without having to change to a new name)